tests/pipeline/test_pipeline.py

```diff
@@ -272,7 +272,9 @@ def test_no_user_defined_components(self, snapshot: SnapshotTest):
         enriched_pipeline: dict = yaml.safe_load(result.stdout)
         snapshot.assert_match(enriched_pipeline, "test-pipeline")
 
-    def test_kafka_connect_sink_weave_from_topics(self, snapshot: SnapshotTest):
+    def test_kafka_connect_sink_weave_from_topics(
+        self, snapshot: SnapshotTest
+    ):  # INTERFERES WITH test_with_env_defaults
```

Comment:
is this fixed?

Reply:
We never figured out why it happened, but I believe it doesn't occur anymore.

Reply:
could you remove the comment then?